TTHAnalysis/python/tools/nanoAOD/ttH_modules.py

```diff
@@ -229,6 +229,7 @@ def _fires(ev, path):
 
 BDThttTT_Hj = lambda : BDT_eventReco(os.environ["CMSSW_BASE"]+'/src/CMGTools/TTHAnalysis/data/kinMVA/tth/TMVAClassification_bloose_BDTG.weights.xml',
                                      os.environ["CMSSW_BASE"]+'/src/CMGTools/TTHAnalysis/data/kinMVA/tth/TMVAClassification_btight_BDTG.weights.xml',
+#                                    os.environ["CMSSW_BASE"]+'/src/CMGTools/TTHAnalysis/data/kinMVA/tth/Hj_2017_configA_dcsv_BDTG.weights.xml',
```

Comment:
What is this?

Reply:
one of the recent pulls from Sergio added the Hjtagger_legacy_xgboost. Resolving conflict I commented this line but it can be removed, should I?

Reply:
@vischia

Reply:
Let's hear from @sscruz I guess